release.sh

```diff
@@ -0,0 +1,91 @@
+set -e
+
+check_version () {
+	if ! [[ $1 =~ ^v[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]
+	then
+	    echo -e "\nThat version does not match semver"
+	    exit
```

Comment:
Maybe we should just return 0 (success) or !=0 (e.g., 1; failure), so the caller can offer the user to enter the tag again.